sdk/servicebus/azservicebus/linkCommon.go

```diff
@@ -0,0 +1,36 @@
+package azservicebus
+
+import "context"
+
+type linkState struct {
```

Comment:
Comment describing what this is used for?

Reply:
Comment added.